sdk/src/main/java/com/mapbox/maps/extension/observable/resourcerequest/eventdata/RequestDataSourceType.kt

```diff
@@ -0,0 +1,38 @@
+package com.mapbox.maps.extension.observable.resourcerequest.eventdata
+
+import com.google.gson.annotations.SerializedName
+
+/**
+ * Describes data source of request for resource-request event.
+ */
+enum class RequestDataSourceType {
```

Comment:
nit: `RequestDataSourceType` doesn't belong to request, we could rename it to `DataSourceType`